tests/e2e_test.go

```diff
@@ -295,6 +300,39 @@ var _ = ginkgo.BeforeSuite(func() {
 	}
 	log.Info("Finished deploying Teleporter contracts")
 
+	// Deploy Native Token bridge on the two subnets
+	{
+		nativeTokenBridgeDeployerPK, err = crypto.HexToECDSA(bridgeDeployerKeyStr)
+		Expect(err).Should(BeNil())
+
+		cmd := exec.Command(
```

Comment:
If it's feasible, we should prefer deploying the contracts using the existing Go RPC clients, rather than spinning up an external call to `forge`. I think it would look something like this:
- Read in the ABI and bytecode from the `forge build` artifacts generated in `e2e_test.sh`
- Deploy the contract using `bind.DeployContract`
  - definition: https://github.com/ava-labs/subnet-evm/blob/3c8709f40c9c8c65a1b23331344dc3fe8c96a303/accounts/abi/bind/base.go#L146
  - example usage: https://github.com/ava-labs/subnet-evm/blob/3c8709f40c9c8c65a1b23331344dc3fe8c96a303/accounts/abi/bind/backends/simulated_test.go#L1025
  - I believe `ethclient.Client` implements `bind.ContractBackend` so we can just pass those clients in directly.

Reply:
Yep! Was finally able to refactor these. Thanks for the suggestion.